OracleLinux/7/scripts/docker-engine.sh

```diff
@@ -0,0 +1 @@
+../../../DockerEngine/scripts/install.sh
```

Comment:
I would copy the script into this location so that we can remove the `DockerEngine` and `LAMP` directories completely in a future pull request. I'd also rename this to `container-engine.sh`

Reply:
I inverted the symlinks (to keep file history)